<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayFairCipher</class>
 <widget class="QMainWindow" name="PlayFairCipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayFairCipher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_GenerateMatrix">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1050</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Matrix</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_PlainText_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>940</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Encrypt_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>1050</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Decrypt_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>1050</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>970</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plain Text:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GenerateMatrix_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1050</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Matrix</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_PlainText_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>940</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Encrypt_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>1050</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Decrypt_3">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>1050</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>970</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plain Text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable4_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generate Matrix:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FAIRPLAY CIPHER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_CipherText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_GenerateMatrix">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Generate_Matrix">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Matrix</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_PlainText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Encrypt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_Key">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Decrypt">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plain Text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cipher Text:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
